app/home/dashboard.njk

```diff
@@ -15,8 +15,26 @@
     {% for key, section in sections %}
       {% if canAccess(section.permission) %}
         <section class="govuk-grid-column-one-half govuk-!-margin-bottom-6">
-          <h2 class="govuk-heading-m govuk-!-margin-bottom-3">
+          <h2 class="govuk-heading-m govuk-!-margin-bottom-1">
             {{ t(section.heading) }}
+
+            <span class="govuk-!-display-block">
+            {% if section.period === 'week' %}
+              {{ dateRange | formatDateRange }}
+              {{ appTime({
+                datetime: dateRange | formatISOWeek,
+                text: '(' + dateRange | formatDateRangeAsRelativeWeek | lower + ')',
+                classes: 'app-secondary-text-colour'
+                }) }}
+            {% else %}
+              {{ today | formatDateWithDay }}
+              {{ appTime({
```

Comment:
So it looks like we've lost the day from the original format and it now also displays the date in grey on every page:

this
![image](https://user-images.githubusercontent.com/3327997/89623511-055e1500-d895-11ea-8e62-e1631db8e0de.png)

should be this:
![image](https://user-images.githubusercontent.com/3327997/89623677-5837cc80-d895-11ea-99d6-1dff28f04d57.png)

and this:
![image](https://user-images.githubusercontent.com/3327997/89623722-6980d900-d895-11ea-8335-b69b6e1fe8ee.png)

should be this:
![image](https://user-images.githubusercontent.com/3327997/89623754-77365e80-d895-11ea-8769-ee56acd41e0a.png)